subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/plugins/JacocoReportAggregationPlugin.java

```diff
@@ -63,35 +60,34 @@ public abstract class JacocoReportAggregationPlugin implements Plugin<Project> {
 
     @Override
     public void apply(Project project) {
-        project.getPluginManager().apply("org.gradle.reporting-base");
```

Comment:
I think we should still apply this, even if it's applied by jacoco too because we directly use extensions from this plugin.

Reply:
OK, will revert.